src/main/java/com/fvv/bookstore/view/BookViewImpl.java

```diff
@@ -0,0 +1,184 @@
+package com.fvv.bookstore.view;
+
+import java.util.List;
+
+import javax.swing.JOptionPane;
+
+import com.fvv.bookstore.bean.Book;
+import com.fvv.bookstore.controller.BookController;
+import com.fvv.bookstore.controller.BookControllerImpl;
+import com.fvv.bookstore.exception.BookValidationException;
+import com.fvv.bookstore.exception.ControllerException;
+
+/**
+ * BookView class to view the Book object.
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 24 de mar de 2018</p>	
+ * @version 1.0 	
+ *
+ */
+public class BookViewImpl implements BookView {
+	
+	private BookController bookController;
+	
+	/**
+	 * Class constructor instantiating a new BookControllerImpl object.
+	 */
+	public BookViewImpl() {
+		this.bookController = new BookControllerImpl();
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public void addBook() {
+		try {
+			Book book = new Book();
+			book.setTitle(JOptionPane.showInputDialog("Insert the title"));
+			book.setPublicationYear(
+					Integer.parseInt(JOptionPane.showInputDialog("Insert the publication year")));
+			book.setEditionNumber(
+					Integer.parseInt(JOptionPane.showInputDialog("Insert the edition number")));
+			book.setAuthor(JOptionPane.showInputDialog("Insert the author"));
+			book.setPrice(
+					Double.parseDouble(JOptionPane.showInputDialog("Insert the price")));
+			book.setIsbn(
+					Integer.parseInt(JOptionPane.showInputDialog("Insert the isbn")));
+			book.setPublisher(JOptionPane.showInputDialog("Insert the publisher"));
+			book.setGenre(JOptionPane.showInputDialog("Insert the genre"));
+			
+			this.bookController.validateBook(book);
```

Comment:
as stated earlier, this should not be here, you need to call this validation in the controller class